### DISCUSSION TITLE
Why is `waitForSelector` an invalid snapshot option?

### DISCUSSION THREAD

**Discussion by JodiWarren** [percy/percy-storybook#618] [Support] [answered]
Within our `.percy.yml` we have the `waitForSelector` option set like so:

```yaml
storybook:
  waitForSelector: '#root > *'
```

When running percy, we receive this error on each snapshot:
```
[percy] Invalid snapshot options:
[percy] - waitForSelector: not accepted with DOM snapshots
```

According to the docs this seems a valid configuration: https://docs.percy.io/docs/storybook#percy-config-file-options

Does anyone know what the problem is?

**Comment by wwilsman** [comment, marked as answer]
Hi @JodiWarren!

It looks like this is a bug with recent changes. The snapshot option is accepted, but that is being incorrectly logged after the option is already used when taking the DOM snapshot. I'll have a PR to fix this later today, but you should be safe to ignore that warning and continue using `waitForSelector`.

**Reply by JodiWarren** [reply]
Thank you! Reassuring to know that it's still taking effect.
